src/z3c/pt/tests/test_templates.py

```diff
@@ -118,19 +124,107 @@ def render(self):
                 implementedBy(Context),
                 implementedBy(Request),
                 implementedBy(View)
-                ),
+            ),
             IContentProvider,
             name="content"
-            )
+        )
 
         context = Context()
         request = Request()
 
         result = view(context=context, request=request)
-        self.assertTrue(repr(data) in result)
-        self.assertTrue(repr({'context': context}) in result)
+        self.assertIn(repr(data), result)
+        self.assertIn(repr({'context': context}), result)
+
+class TestOpaqueDict(unittest.TestCase):
+
+    def test_getitem(self):
+        import operator
+        d = {}
+        od = pagetemplate.OpaqueDict(d)
+        with self.assertRaises(KeyError):
+            operator.itemgetter('key')(od)
```

Comment:
Why this cumbersome spelling of `od['key']`?  It looks like a mechanical translation of
`self.assertRaises(KeyError, operator.itemgetter('key'), od)`.

Reply:
Because just saying `od['key']` causes the linter to warn that it's a useless statement (because it's not assigned to anything). And assigning it to something (`_ = od['key']`) causes the linter to warn that the new variable is unused.

Reply:
Therefore pylint is evil. Q.E.D.